.github/workflows/connect-dev-release.yml

```diff
@@ -18,6 +18,7 @@ on:
       - "packages/utils"
       - "submodules/trezor-common"
       - "yarn.lock"
+      - ".github/workflows/connect-dev-release.yml"
```

Comment:
This is a good point, run it also when the test files are changed. We should use it in all the tests. :+1: